FirstProject/npm-debug.log

```diff
@@ -0,0 +1,25 @@
+0 info it worked if it ends with ok
```

Comment:
It is in .gitignore..
Is not good written?

![gitig](https://user-images.githubusercontent.com/31060543/29554370-83924378-8727-11e7-9a0b-0f8b5a8da84a.png)